lxd/instance/drivers/driver_qemu.go

```diff
@@ -4023,11 +4023,50 @@ func (d *qemu) Update(args db.InstanceArgs, userRequested bool) error {
 
 	if isRunning {
 		// Only certain keys can be changed on a running VM.
-		liveUpdateKeys := []string{"limits.memory"}
+		liveUpdateKeys := []string{
+			"limits.memory",
+			"security.agent.metrics",
+		}
+
+		isLiveUpdatable := func(key string) bool {
```

Comment:
Which of these were required for VM live migration?

Is any of this logic in common with containers. And if so can we indicate this somehow via a common function ?

Reply:
In this case, I needed one of the volatile keys for this.
Containers have no such checks, all keys are always editable with some requiring a reboot to fully apply.

The plan for VMs was to get as close to parity with containers as possible and then align behaviors, the current errors are mostly there to highlight that things someone may think will work because they work on containers will not work on a VM.